scripts/test

```diff
@@ -2,11 +2,7 @@
 
 set -euf -o pipefail
 
-source "${BASH_SOURCE%/*}/../node_modules/sanctuary-scripts/functions"
-
-branches="$(get min-branch-coverage)"
-
 npm run licensetest
```

Comment:
Would you like sanctuary-scripts to support this?

Reply:
I do not think it is general enough for being included in sanctuary-scripts.